go/teams/loader2.go

```diff
@@ -525,6 +531,84 @@ func (l *TeamLoader) checkProofs(ctx context.Context,
 	return proofSet.check(ctx, l.world)
 }
 
+func (l *TeamLoader) unboxKBFSCryptKeys(ctx context.Context, key keybase1.TeamApplicationKey,
+	keysetHash string, encryptedKeyset string) ([]keybase1.CryptKey, error) {
+
+	// Check hash
+	sbytes := sha256.Sum256([]byte(encryptedKeyset))
+	if hex.EncodeToString(sbytes[:]) != keysetHash {
```

Comment:
Can you use secure equal here? Always when comparing macs/hashes, it's a good idea.

Reply:
Good point